lib/methods/template.js

```diff
@@ -75,7 +75,12 @@ const splitByWhitespaces = (template, rawTemplate) => {
 			templateStart = templateIndex + 1;
 		} else if (rawCharacter === '\\') {
 			const nextRawCharacter = rawTemplate[rawIndex + 1];
-			if (nextRawCharacter === 'u' && rawTemplate[rawIndex + 2] === '{') {
+			if (nextRawCharacter === '\n') {
+				// Handles escaped newlines in templates
+				templateIndex -= 1;
+				rawIndex += 1;
+				continue;
```

Comment:
This `continue` is probably going to make somebody crabby, but my intention here is to explicitly indicate that I'm doing weird stuff to the control flow of the loop because I'm mutating `templateIndex` and putting it temporarily into an invalid state, anticipating that the for loop will re-increment it again resulting in the value remaining stable while the newlines are skipped.

Reply:
Yes, I think the code is clear enough without it, and the `continue` here is a noop. 

We have very thorough testing for that file, so if we were to add more logic after those lines in the future, we'd catch any potential problem.

Based on this, I think this `continue` should be removed.